docs/onboarding/rpc/node.md

```diff
@@ -131,15 +131,6 @@ Gets (and optionally streams) the node's status
 }
 `} />
 
-import Terminal from '../../../src/theme/components/Terminal/Terminal'
-import GetStatus from '../../../src/theme/components/Terminal/rpc/GetStatus'
-
-```
-curl -X POST http://localhost:8021/node/getStatus
-```
-
-<Terminal command={GetStatus} />
```

Comment:
Can you delete this GetStatus file too? Or is it used elsewhere

Reply:
deleted